README.md

```diff
@@ -1,131 +1,2469 @@
-# go-nautobot
+# Go API client for nautobot
 
-## :warning: Disclaimer :warning:
```

Comment:
I want to keep the disclaimer until the project is more mature

Reply:
Done :)